lib/rules/media-feature-name-value-whitelist/README.md

```diff
@@ -0,0 +1,65 @@
+# media-feature-name-value-whitelist
+
+Specify a whitelist of allowed media feature and value pairs.
+
+```css
+@media screen and (min-width: 768px) {}
+/**                    ↑        ↑
```

Comment:
Please align the arrows with the start of each bit of vocab i.e. the "m" in "min-width" and "7" in "768px"

(There are a lot of rules in stylelint, so we try to be consistent as possible across all the docs).

Reply:
Will do. I've not come across this in the docs so I wasn't aware of this rule.

Reply:
Yep, it's not an obvious one to glean. I've [created a PR](https://github.com/stylelint/stylelint/pull/3363) to explicitly highlight the convention (along with the "This rule ..." one).